### PR TITLE
Add new collaborators and change permissions on yjaf* repos

### PR DESCRIPTION
**Pull request by AntonyBishop**
Adding collaborators and changing permissions to allow collaborators access to Dependabot alerts and issues.

### CODE REVIEW

File: terraform/YJAF-POC-APP.tf [hunk 1]
```diff
@@ -52,5 +52,25 @@ module "YJAF-POC-APP" {
...
+      review_after = "2022-08-27"
+    },
+    {
+      github_user  = "richardbradwell"
+      permission   = "admin"
+      name         = "Richard Bradwell"
+      email        = "richard.bradwell@necsws.com"
+      org          = "NEC Software Solutions"
+      reason       = "YJAF developer. Needing access to the work on BackEnd coding Tasks"
+      added_by     = "Jo Harvey <joanna.harvey@necsws.com"
```
Comment (NickWalt01) [inline terraform/YJAF-POC-APP.tf:72]: I think this line will fail as it not a justice email

Comment (AntonyBishop) [reply terraform/YJAF-POC-APP.tf:72]: And it did. I'll go in and change update them. Should just be for the new additions.

Comment (AntonyBishop) [reply terraform/YJAF-POC-APP.tf]: added_by values updated.

**Comment by AntonyBishop** [comment]
Pick the checks

**Review by NickWalt01 — APPROVED** [review]

**Pull request merged**